packages/picker/src/Picker.ts

```diff
@@ -539,7 +539,9 @@ export class PickerBase extends SizedMixin(Focusable) {
                 this.menuItems = this.optionsMenu.childItems;
             } else {
                 this.menuItems = [
-                    ...this.querySelectorAll('sp-menu-item'),
+                    ...this.querySelectorAll(
```

Comment:
Did you end up liking this more than the gating? There's _something_ to be said about a single selection Picker that has submenus...but I'm not sure that was accurately covered previously, either.

Reply:
Personally I like the gating more (simpler conceptually - something with no `selects` behavior doesn't run `manageSelection`), but at the root level, items are selectable by default. With the gate:

![image](https://user-images.githubusercontent.com/364501/182927483-0600e680-87f7-4c47-b9da-5cbd1fc176ea.png)

Reply:
With the selector approach, can you confirm that this works there, too? https://studio.webcomponents.dev/edit/guTpKeAjgecibF150Qbg/src/index.ts?branch=group%40wfVpXdsTYhf9mY2slxW4t2Y2SjC3&p=stories Might just need some massage to get it working, but want to ensure we're not regressing on anything that might not be accurately covered in the tests.

Reply:
Good call, take a look at the latest commit - I added that sketch as a test so we can be sure to not regress it, & loosened up the query a little to support it.

Reply:
That's a fancy selection. I like it. Don't usually use `:not()` in that way, but looks powerful. :clap: